### ISSUE TITLE
Samsung Galaxy S5 Xposed installation crashes

### ISSUE THREAD

**Issue by gjohnson5005** [rovo89/Xposed#129]
I've been trying to re-install Xposed but can't seem to get it to work.  It worked fine with 4.4.2 but with 5.0, nothing seems to work.  Have re-rooted, rebuilt several times with no joy.  I've dropped a logcat.  Hope someone can help me out.
[logcat1.txt](https://github.com/rovo89/Xposed/files/129966/logcat1.txt)

**Comment by rovo89** [comment]
Make sure to read the red text here: http://forum.xda-developers.com/showthread.php?t=3034811
Please check the same subforum for unofficial versions that support Samsung. See also http://forum.xda-developers.com/showpost.php?p=58829021&postcount=2

**Comment by gjohnson5005** [comment]
Thanks for the quick response.  I've read the above links and have wiped
and restored more times than I care to think about.  I just tried a fresh
load with a bare minimum of apps.  Re-rooted, loaded TWRP.  I tried loading
the art97 load and got an error about not being deoxed.  Tried loading the
sdk 21 armv7-v80 load and the screen is stuck again at the Samsung logo.
Not really sure what to try next except to not install xposed.  Any further
suggestions?

Thanks in advance.
